stable/artifactory/Chart.yaml

```diff
@@ -1,7 +1,11 @@
 apiVersion: v1
 name: artifactory
 home: https://www.jfrog.com/artifactory/
+<<<<<<< HEAD
```

Comment:
@danielezer same here

Reply:
Whoops, sorry I didn't commit my conflict resolutions. Sorry about that